### ISSUE TITLE
wrong m3u after second scan of a library

### ISSUE THREAD

**Issue by lopatkovn** [teticio/Deej-AI#69]
Hi , got a bug :
Generated database with command (20 of 20 has scanned)
python MP3ToVec.py Pickles mp3tovec --scan C:\mp3library
Than generated m3u with
python Deej-A.I.py Pickles mp3tovec --playlist C:\m3ufiles\000000018.m3u8 --inputsong C:\mp3library\66.mp3 --nsongs 3
Got correct m3u ( with similar songs)
Next added another song to C:\mp3library , and ran the command again
python MP3ToVec.py Pickles mp3tovec --scan C:\mp3library
New song added to database (1 of 1 has scanned)

When tried to get m3u again 
python Deej-A.I.py Pickles mp3tovec --playlist C:\m3ufiles\000000018.m3u8 --inputsong C:\mp3library\66.mp3 --nsongs 3
got error :
C:\Deej-AI-master\Deej-A.I.py:186: RuntimeWarning: invalid value encountered in scalar divide
  cos_proximity = np.dot(mp3_vec_i, mp3_vec_j) / (np.linalg.norm(mp3_vec_i) * np.linalg.norm(mp3_vec_j))
Got m3u generated , but songs in it not similar at all

In short - after adding new song to lib and additional scan , that leads to error (with wrong(not similar) m3u output)

**Comment by lopatkovn** [comment]
Additional info :
If i remove only mp3tovecs.p from Pickles\mp3tovecs , and do rescan , then newly generated m3u file contains right(similar to input) songs  ,and im not getting (RuntimeWarning: invalid value encountered in scalar divide).
Proccess of rescan goes faster (without rescanning songs , only vectors) , but still not fast enough for big library (1-2 hours).

more in short - additional scan after adding new song breaks mp3tovecs.p file  ,that leads to wrong generated m3u

**Comment by lopatkovn** [comment]
add : this happens when adding only 1 song and run scan . 2 will not lead to this.
